sidecar-injector/README.md

```diff
@@ -126,61 +242,272 @@ For this section, you'll work from a test namespace (test-namespace). Later you
+    ```
+
+1. Create test namespace
+    ```bash
+    ~$ kubectl create namespace ${TEST_APP_NAMESPACE_NAME}
+    ```
+
+1. Label the default namespace with `cyberark-sidecar-injector=enabled`
+    ```bash
+    ~$ kubectl label namespace ${TEST_APP_NAMESPACE_NAME} cyberark-sidecar-injector=enabled
+    ~$ kubectl get namespace -L cyberark-sidecar-injector
+    ```
+    ```
+    NAME                            STATUS    AGE       CYBERARK-SIDECAR-INJECTOR
+    default                         Active    18h
+    kube-public                     Active    18h
+    kube-system                     Active    18h
+    cyberark-sidecar-injector       Active    18h
+    sidecar-example-app             Active    18h       enabled
+    ```
+
+1. Create service account
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} create serviceaccount ${TEST_APP_SERVICE_ACCOUNT}
+    ```
+
+1. Create Conjur ConfigMap
+    ```bash
+    ~$ cat << EOL | kubectl -n ${TEST_APP_NAMESPACE_NAME} apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: conjur
+    data:
+      CONJUR_ACCOUNT: "${CONJUR_ACCOUNT}"
+      CONJUR_VERSION: "${CONJUR_VERSION}"
+      CONJUR_APPLIANCE_URL: "${CONJUR_APPLIANCE_URL}"
+      CONJUR_AUTHN_URL: "${CONJUR_AUTHN_URL}"
+      CONJUR_SSL_CERTIFICATE: |
+    $(echo "${CONJUR_SSL_CERTIFICATE}" | awk '{ print "    " $0 }')
+      CONJUR_AUTHN_LOGIN: "${CONJUR_AUTHN_LOGIN}"
     EOL
     ```
+
+### Deploy Authenticator Sidecar
+
+1. Deploy an app with the Authenticator Sidecar, take `test-app` app as an example
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} delete pod test-app
+    ~$ cat << EOF | kubectl -n ${TEST_APP_NAMESPACE_NAME} apply -f -
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      annotations:
+        sidecar-injector.cyberark.com/conjurAuthConfig: conjur
+        sidecar-injector.cyberark.com/conjurConnConfig: conjur
+        sidecar-injector.cyberark.com/containerMode: ${containerMode}
+        sidecar-injector.cyberark.com/inject: "yes"
+        sidecar-injector.cyberark.com/injectType: authenticator
+        sidecar-injector.cyberark.com/containerName: secretless
+      labels:
+        app: test-app
+      name: test-app
+    spec:
+      containers:
+      - image: googlecontainer/echoserver:1.1
+        name: app
+        volumeMounts:
+        - mountPath: /run/conjur
+          name: conjur-access-token
+      serviceAccountName: ${TEST_APP_SERVICE_ACCOUNT}
+    EOF
+    ```
+
+1. Verify Authenticator sidecar container injected
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} get pods
+    ```
     ```
-    "test-secret#username:test-secret#password"
+    NAME                     READY     STATUS        RESTARTS   AGE
+    test-app                 2/2       Running       0          1m
+    ```
+
+1. Test Authenticator
```

Comment:
Getting an error here:

```
cat: can't open '/run/conjur/access-token': No such file or directory
```

Reply:
I'm still getting this error. Pretty sure I followed all the steps correctly, but I must still be missing something.